pallets/rmrk-core/src/lib.rs

```diff
@@ -490,37 +491,19 @@ pub mod pallet {
 			license: Option<BoundedVec<u8, T::StringLimit>>,
 			thumb: Option<BoundedVec<u8, T::StringLimit>>,
 		) -> DispatchResult {
-			let sender = Self::ensure_protocol_or_signed(origin)?;
-
-			let mut pending = false;
-			let nft = NFTs::<T>::get(collection_id, nft_id).ok_or(Error::<T>::NoAvailableNftId)?;
-			if nft.rootowner != sender.unwrap_or_default() {
-				pending = true;
-			}
-
-			let resource_id = Self::get_next_resource_id()?;
-			ensure!(
-				Resources::<T>::get((collection_id, nft_id, resource_id)).is_none(),
-				Error::<T>::ResourceAlreadyExists
-			);
+			let sender = Self::ensure_protocol_or_signed(origin)?.unwrap_or_default();
```

Comment:
please pull `main` 
should be simply `let sender = ensure_signed(origin.clone())?;`

Reply:
resolved in https://github.com/rmrk-team/rmrk-substrate/pull/41/commits/e8c89c2346bbfcb3104727e6bb296de4646b619f